app/src/main/java/org/mozilla/reference/browser/BrowserActivity.kt

```diff
@@ -120,6 +121,13 @@ open class BrowserActivity : AppCompatActivity() {
             else -> super.onCreateView(parent, name, context, attrs)
         }
 
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        Logger.info("Activity onActivityResult received with " +
+            "requestCode: $requestCode, resultCode: $resultCode, data: $data")
```

Comment:
nit: Is OK to log this data? Is not sensible data?

Reply:
For debugging, I need to be able to see the activity result that's coming through. Once the feature works, we can remove this.

Another reason I didn't want to commit this logging code to Fenix. :)